components/resources/ai_chat_ui_strings.grdp

```diff
@@ -235,6 +235,9 @@
   <message name="IDS_CHAT_UI_PAGE_CONTENT_REFINED_WARNING" desc="A warning issued when the page content has been refined">
     The page content has been refined through extracting the most relevant information from the page because it was too long for Leo.
   </message>
+  <message name="IDS_CHAT_UI_PAGE_CONTENT_REFINED_IN_PROGRESS" desc="An indicator to show page content refined is in progress">
+    Page content refining in progress...
```

Comment:
@petemill do we need `is`?

Reply:
Added in 0b4ee1e1e116f7b2224ed02c99dc5198de4677c7.